Gemfile

```diff
@@ -75,7 +76,7 @@ gem 'rsolr'
 gem 'rspec-rails'
 gem 'rubyzip', '>= 1.2.2'
 # Use SCSS for stylesheets
-gem 'sass-rails', '~> 6.0'
```

Comment:
Could we remove this outright, rather than commenting it?

Reply:
yes I will remove it. I will add it in the commit for the Gemfile. I want to keep them tidy and clean.

Reply:
Thanks for looking into the skip links. I will check what you pointed out.